loader/loader.go

```diff
@@ -1295,18 +1273,16 @@ func (l *Loader) cleanDumpFiles() {
 			l.logCtx.L().Warn("error when remove loaded dump folder", zap.String("data folder", l.cfg.Dir), zap.Error(err))
 		}
 	} else {
-		// leave metadata file, only delete structure files
-		for db, tables := range l.db2Tables {
-			dbFile := fmt.Sprintf("%s/%s-schema-create.sql", l.cfg.Dir, db)
-			if err := os.Remove(dbFile); err != nil {
-				l.logCtx.L().Warn("error when remove loaded dump file", zap.String("data file", dbFile), zap.Error(err))
-			}
-			for table := range tables {
-				tableFile := fmt.Sprintf("%s/%s.%s-schema.sql", l.cfg.Dir, db, table)
-				if err := os.Remove(tableFile); err != nil {
-					l.logCtx.L().Warn("error when remove loaded dump file", zap.String("data file", tableFile), zap.Error(err))
-				}
+		// leave metadata file, only delete sql files
+		files := CollectDirFiles(l.cfg.Dir)
+		var lastErr error
+		for f := range files {
+			if strings.HasSuffix(f, ".sql") {
+				lastErr = os.Remove(f)
 			}
 		}
+		if lastErr != nil {
+			l.logCtx.L().Warn("error when remove loaded dump sql files", zap.String("data folder", l.cfg.Dir), zap.Errors("last error", []error{lastErr}))
```

Comment:
why do you use `zap.Errors` with only one error?

Reply:
https://github.com/pingcap/dm/pull/1027/commits/3a629dd9115857c6d47733a573342e7f0f6fd306